lib/vehicle.rb

```diff
@@ -22,4 +27,13 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def registered?    
+    !!plate_type && !!registration_date
+  end
```

Comment:
I'm curious about this method. I didn't see anywhere in the interaction pattern where you needed to create this method, and it doesn't look like it's being used as a helper method anywhere else. You could get rid of it. 

That said, as a general practice, it doesn't make too much sense to throw a double bang (!!) in front of something, since that essentially negates the negation (like a double negative), so it just makes it what it was originally.